petab/math/SympyVisitor.py

```diff
@@ -35,8 +35,8 @@
 }
 _unary_funcs = {
     "exp": sp.exp,
-    "log10": lambda x: sp.log(x, 10),
-    "log2": lambda x: sp.log(x, 2),
+    "log10": lambda x: -sp.oo if x.is_zero is True else sp.log(x, 10),
+    "log2": lambda x: -sp.oo if x.is_zero is True else sp.log(x, 2),
```

Comment:
```suggestion
    "log10": lambda x: -sp.oo if x.is_zero else sp.log(x, 10),
    "log2": lambda x: -sp.oo if x.is_zero else sp.log(x, 2),
```

Reply:
I prefer the `is` expressions for all `is_*` attributes, since they aren't bool, but can also be None. For `is True`, this doesn't matter so much, but it feels safer if somebody turns the expression around.